src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -6,27 +6,29 @@
 
 #pragma once
```

Comment:
In commit message typo "Dirgraph.hpp" -> "Digraph.hpp"

Reply:
Fixed.